packages/core/components/package.json

```diff
@@ -11,7 +11,8 @@
     "build": "parcel build"
   },
   "publishConfig": {
-    "access": "public"
+    "access": "public",
+    "group": "@propeldata/ui-kit"
```

Comment:
All our packages will be considered part of a group named "@propeldata/ui-kit". They will all share the same version (Lerna-style). The resulting git tags will look like "@propeldata/ui-kit@x.y.z" and "@propeldata/ui-kit@x.y.z-rc.N".